libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -369,6 +369,26 @@ bool AP_Arming::gps_checks(bool report)
             return false;
         }
     }
+
+    // check GPSs are within 50m of each other and that blending is healthy
+    if ((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_GPS_CONFIG)) {
+        float distance_m;
+        if (!gps.all_consistent(distance_m)) {
+            if (report) {
+                GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL,
+                                                 "PreArm: GPSs inconsistent by %4.1fm",
```

Comment:
One issue with making long descriptions is that, at least on the mission planner, most users will view them on the HUD which is not very wide. The message is already at it's maximum length so any additional characters are simply at the expense of characters at the end of the message.
In any case, I've changed it to "PreArm: GPS positions differ by %4.1fm"